rules/creating-action-items/rule.md

```diff
@@ -9,15 +9,25 @@ related:
   - do-you-know-what-happens-at-a-sprint-review-meeting
 redirects:
   - share-the-action-items-that-came-up
+  - follow-up-effectively
+  - stick-to-the-agenda-and-complete-the-meetings-goal
 created: 2021-07-20T06:35:55.000Z
 archivedreason: null
 guid: 50f5fd0b-a15f-45ad-bb35-2dd522d9a5db
 ---
 The aim of most meetings should be to come up with next steps. These should be shared with the intended recipient, as well as CCing all other attendees.
 
-It is a good idea to have a scribe who is drafting the action points during the meeting, so if it's not already clear who that is, say "Who will be the scribe?" at the beginning of any meeting.
+It is a good idea to have a scribe who is drafting the action points during the meeting and **assigning them directly to those resposible for completing the tasks**. If it's not already clear who the scribe is, say "Who will be the scribe?" at the beginning of any meeting. And now in Microsoft Teams, when in a meeting, the scribe can use the **Notes** tab to add details and list follow-up tasks during the meeting.
```

Comment:
Drop the `And now`, it won't age well. I feel just saying, In MS Teams we can do... is ok 🙂

```suggestion
It is a good idea to have a scribe who is drafting the action points during the meeting and **assigning them directly to those resposible for completing the tasks**. If it's not already clear who the scribe is, say "Who will be the scribe?" at the beginning of any meeting. In Microsoft Teams, when in a meeting, the scribe can use the **Notes** tab to add details and list follow-up tasks during the meeting.
```